ml-agents-envs/mlagents_envs/environment.py

```diff
@@ -249,7 +249,11 @@ def _executable_args(self) -> List[str]:
         if self._no_graphics:
             args += ["-nographics", "-batchmode"]
         args += [UnityEnvironment._PORT_COMMAND_LINE_ARG, str(self._port)]
-        if self._log_folder:
+
+        # If the logfile arg isn't already set in the env args,
+        # try to set it to an output directory
+        logfile_set = "-logfile" in (arg.lower() for arg in self._additional_args)
```

Comment:
This was annoying - if you wanted to pass e.g. `--env-args -logfile -` to `mlagents-learn` it would get overwritten here.